### ISSUE TITLE
spinner is not getting click in web page

### ISSUE THREAD

**Issue by renas** [RobotiumTech/robotium#683]
_From [prabji...@gmail.com](https://code.google.com/u/107776071599865102289/) on May 12, 2014 04:43:17_

What steps will reproduce the problem? 1.spinner is there on webpage
2.enable to click on the spinner
3.tried all the web element of spinner still not working What is the expected output? What do you see instead? web element of spinner is not found. What version of the product are you using? On what operating system? robotium.solo 5.1 and android device 4.3 Please provide any additional information below. I am testing a web site other options are working fine with the test the new problem occur spinner is not getting clicked and opening the list.

_Original issue: http://code.google.com/p/robotium/issues/detail?id=604_

**Comment by renas** [comment]
_From [prabji...@gmail.com](https://code.google.com/u/107776071599865102289/) on May 12, 2014 06:20:12_

please help me i am stuck

**Comment by renas** [comment]
_From [renasr...@gmail.com](https://code.google.com/u/108052454046732184536/) on May 12, 2014 08:52:33_

Thanks for reporting this. Would it be possible for you to share an app that exhibits this issue?

**Owner:** renasr...@gmail.com

**Comment by renas** [comment]
_From [prabji...@gmail.com](https://code.google.com/u/107776071599865102289/) on May 12, 2014 22:23:24_

cannot share the code but i will explain you on the webpage there is one spinner now i want to click the spinner select the value and move to next page for clicking the spinner i using solo.clickOnWebElement(By.xpath("//div[@id\='surveyquescontent']/span[2]/select/option[4]"));
i have also tried  solo.clickOnWebElement(By.className("mrDropdown"));
solo.clickOnWebElement(By.id("_Q0_C"), 0, true);

**Comment by renas** [comment]
_From [renasr...@gmail.com](https://code.google.com/u/108052454046732184536/) on June 08, 2014 07:07:24_

Please try to set config to: conf.useJavaScriptToClickWebElements = true. Please see the javaodoc for more information.

**Status:** WontFix

**Comment by renas** [comment]
_From [prabji...@gmail.com](https://code.google.com/u/107776071599865102289/) on June 09, 2014 03:11:28_

Config config = new Config();
		config.useJavaScriptToClickWebElements = true;
		config.screenshotFileType = ScreenshotFileType.PNG;
		config.screenshotSavePath = Environment.getExternalStorageDirectory()
				+ "/Robotium/";
		config.shouldScroll = true;
		solo = new Solo(getInstrumentation(), config, getActivity());
		
already added this line but still not working


i Have

**Comment by renas** [comment]
_From [renasr...@gmail.com](https://code.google.com/u/108052454046732184536/) on June 09, 2014 03:12:38_

Can you share an app that exhibits this issue? A simple test app is enough.

**Comment by renas** [comment]
_From [prabji...@gmail.com](https://code.google.com/u/107776071599865102289/) on June 09, 2014 03:13:52_

ok i will try....

**Comment by renas** [comment]
_From [bsztu...@epitiro.com](https://code.google.com/u/100231105112351715139/) on July 21, 2014 07:44:47_

I am also having the same issue. I have used the click method on other objects suck as buttons, text boxes and other things. The main issue is with spinners I have tried to identify them using name/id and xpath. I think that the problem could be when you click on the spinner itself the type is called select-one which implies that it is a html element. However on android it becomes a spinner.
